src/constants.cr

```diff
@@ -25,6 +25,11 @@ module PlaceOS
   AUTH_HOST        = ENV["PLACE_AUTH_HOST"]? || "auth"
   METRICS_ROUTE    = ENV["PLACE_METRICS_ROUTE"]? || "monitor"
 
+  # Backoffice
+
+  INTERFACE_BRANCH = ENV["PLACE_BACKOFFICE_BRANCH"]?.presence || "build/#{production? ? "prod" : "dev"}"
+  INTERFACE_COMMIT = ENV["PLACE_BACKOFFICE_COMMIT"]?.presence || "HEAD"
```

Comment:
May be a little neater to match these constant end env var names (with the `PLACE_` prefix dropped) for consistency.